addons/modules/functions/fnc_moduleCrewToGunner.sqf

```diff
@@ -0,0 +1,49 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Zeus module function to have AI crew take the place of an incapacitated gunner.
+ *
+ * Arguments:
+ * 0: Logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC] call zen_modules_fnc_moduleCrewToGunner
+ *
+ * Public: No
+ */
+
+params ["_logic"];
+
+private _vehicle = attachedTo _logic;
+deleteVehicle _logic;
+
+if (isNull _vehicle) exitWith {
+    [LSTRING(NoObjectSelected)] call EFUNC(common,showMessage);
+};
+
+if (!alive _vehicle) exitWith {
+    [LSTRING(OnlyAlive)] call EFUNC(common,showMessage);
+};
+
+if (fullCrew [_vehicle, "gunner", true] isEqualTo []) exitWith {
+    [LSTRING(MustHaveGunner)] call EFUNC(common,showMessage);
+};
+
+_vehicle allowCrewInImmobile true;
+
+private _gunner = gunner _vehicle;
+private _newGunners = (crew _vehicle - [_gunner]) select {canMove _x};
```

Comment:
Why `canMove` instead of `alive`?

Reply:
I made that change to test against ACE. Of course it doesn't, while `lifeState` does. I made the change in the wrong branch. Sorry!
https://github.com/zen-mod/ZEN/pull/423/commits/e01f8dd92a979d9c67009a4b384b7da4ad07922b